lib/modules/manager/custom/jsonata/schema.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod';
+
+const DepObjectSchema = z.object({
+  currentValue: z.string().optional(),
+  datasource: z.string().optional(),
+  depName: z.string().optional(),
```

Comment:
I think depName is also required?

Reply:
It doesn't need to be there, as we can also configure a `depNameTemplate`